pyproject.toml

```diff
@@ -8,6 +8,7 @@ packages = [{include = "microbial_thermodynamics"}]
 
 [tool.poetry.dependencies]
```

Comment:
It works without doing this but I would add `numpy` as an explicit dependancy (via `poetry add numpy`). At the moment `numpy` is a dependancy of one of your other dependancies, which means you could lose access to it if you removed that dependancy. So it's better to have everything you directly use in the `pyproject.toml` as a dependancy